DaemonRenderer/Source/Include/Resource/IResource.hpp

```diff
@@ -59,7 +59,7 @@ class IResource
          * \param in_manager Resource manager instance. This is useful to request dependencies or resolve assets name/path.
          * \param in_descriptor Resource loading descriptor. This structure can be inherited to pass custom parameters to the loader
          */
-        virtual DAEvoid Load(class ResourceManager& in_manager, class ResourceLoadingDescriptor const& in_descriptor) = 0;
+        virtual DAEvoid Load(class ResourceManager& in_manager, class Renderer& in_renderer, class ResourceLoadingDescriptor const& in_descriptor) = 0;
```

Comment:
The instance of the renderer should be passed in the ResourceLoadingDescriptor instead